### PR TITLE
Publish a transpiled 'lib' directory rather than the current ES6 src

### PR DESCRIPTION
**Pull request by dan-f**
@dmitrizagidulin

### CODE REVIEW

**Comment by dan-f** [comment]
@dmitrizagidulin mind giving this a quick review? the fact that we're currently distributing the es6 source is blocking some current work

File: Makefile [hunk 1]
```diff
@@ -54,7 +58,6 @@ minify: dist/rdflib.min.js
 writable:
 	@sed -i -re 's/git:\/\/github.com\//git@github.com:/' .git/config
 
-.PHONY: test
```
Comment (dmitrizagidulin) [inline Makefile]: I think we should leave the `.PHONY` statements in?

**Comment by dmitrizagidulin** [comment]
Other than the makefile comment, looks good to me!

File: Makefile (hunk 1 above)
Comment (dan-f) [inline Makefile]: I did - I just collected them all at the top, because I find it a bit more readable.

Comment (dmitrizagidulin) [inline Makefile]: I don't mind either way, but I thought the general Makefile convention was to place them right before their respective statements? (vs at the top)

Comment (dan-f) [inline Makefile]: I haven't written Makefiles in a while, so I dunno. I like your suggestion though. I'll add a `.PHONY` on the line prior to each phony target.

**Comment by dmitrizagidulin** [comment]
Looks good. (Thanks for doing this btw! It was making my life difficult as well.)

**Comment by dan-f** [comment]
np! I'll merge and publish a new version once tests pass. :)

**Pull request merged**